packages/backend/src/core/activity/processors/StarkexProcessor.ts

```diff
@@ -25,18 +26,20 @@ export function createStarkexProcessor(
   options: StarkexProcessorOptions,
 ): SequenceProcessor {
   const batchSize = getBatchSizeFromCallsPerMinute(options.singleStarkexCPM)
+  const startDay = options.sinceTimestamp.toStartOf('day').toDays()
 
   return new SequenceProcessor(
     projectId.toString(),
     logger,
     sequenceProcessorRepository,
     {
       batchSize,
-      startFrom: options.sinceTimestamp.toStartOf('day').toDays(),
+      startFrom: startDay,
       // eslint-disable-next-line @typescript-eslint/require-await
       getLatest: async () =>
         getStarkexLastDay(clock.getLastHour(), options.starkexApiDelayHours),
       processRange: async (from, to, trx) => {
+        from = Math.max(from - options.starkexRepeatLastDays, startDay)
```

Comment:
this line is a bit misleading for me, I assume `from` is the amount of days
maybe adding a custom type and explicitly typing it is a solution, I dunno tbh

Reply:
Yeah, I also wouldn't choose this as a solution. It is definitely not intuitive and also affects syncing from scratch as opposed to just getting new values.